services/web/server/src/simcore_service_webserver/garbage_collector/plugin.py

```diff
@@ -32,6 +33,8 @@ def setup_garbage_collector(app: web.Application) -> None:
 
     app.cleanup_ctx.append(run_background_task)
 
+    set_parent_module_log_level(__name__, logging.INFO)
```

Comment:
TIPS
1. Use `_logger.name` instead of `__name__`
2. Do not hard-code INFO but instead use it as minimum LOG level, i.e. this way if we set e.g. WARNING, it will be INFO but if you set DEBUG it remains DEBUG

```python

set_parent_module_log_level(
       _logger.name, 
       min(logging.INFO, get_application_settings(app).WEBSERVER_LOGLEVEL) 
) 

```